apps/docs-snippets/src/guide/introduction/getting-started.test.ts

```diff
@@ -0,0 +1,35 @@
+import { FUEL_NETWORK_URL, Provider, Signer, Wallet } from 'fuels';
+
+/**
+ * @group node
+ * @group browser
+ */
+describe('Getting started', () => {
+  beforeAll(() => {
+    // Avoids using the actual network.
+    vi.spyOn(Provider, 'create')
+      .mockImplementationOnce(
+        () => Provider.create(FUEL_NETWORK_URL)
+      )
+  })
+
+  it('can connect to a local network', async () => {
+    // #region connecting-to-the-local-node
+    // #import { Provider, Wallet, Signer };
+
+    // Create a provider.
+    const LOCAL_FUEL_NETWORK = "http://127.0.0.1:4000/graphql"
+    const provider = await Provider.create(LOCAL_FUEL_NETWORK);
+
+    // Create our wallet (with a private key).
+    const PRIVATE_KEY = 'a1447cd75accc6b71a976fd3401a1f6ce318d27ba660b0315ee6ac347bf39568';
+    const wallet = Wallet.fromPrivateKey(PRIVATE_KEY, provider);
+
+    // Sign a message.
+    const signer = new Signer(PRIVATE_KEY);
+    signer.sign(Buffer.from("Hello, world!"));
```

Comment:
IMO we should keep snippets specific to the context of the doc, maybe create wallet could stay but I think signing is irrelevant here.

Reply:
598953c8a43fbbf06a9a76e1cbbbfb29ead32780